contracts/CrowdFunding.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT 
+pragma solidity ^0.8.28;
+
+contract CrowdFunding {
+    
+    struct Campaign {
+        uint id;
+        address creator;
+        string title;
+        string description;
+        uint goal;
+        uint fundsRaised;
+        uint deadline;
+        bool withdrawn;
+    }
+
+    struct Contribution {
+        address donor;
+        uint projectId;
+        uint amount;
+
```

Comment:
can you remove this space,  pls add linter and prettier to this project, or run `npm run format`, but do it in other PR.

Reply:
Done!